<ui version="4.0" >
 <class>OverrideEditorBase</class>
 <widget class="QWidget" name="OverrideEditorBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Override Editor</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="2" >
    <widget class="QLabel" name="mAttributeLbl" >
     <property name="text" >
      <string>Attributes:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="mPathLbl" >
     <property name="text" >
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QListWidget" name="mMatchesList" />
   </item>
   <item row="0" column="4" >
    <widget class="QToolButton" name="mRemoveBtn" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../../../../MaestroResource.qrc" >:/Maestro/images/edit_remove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3" >
    <widget class="QToolButton" name="mAddBtn" >
     <property name="text" >
      <string>...</string>
     </property>
     <property name="icon" >
      <iconset resource="../../../../MaestroResource.qrc" >:/Maestro/images/edit_add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="mPathCB" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3" >
    <widget class="QTableView" name="mOverrideTableView" />
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../MaestroResource.qrc" />
 </resources>
 <connections/>
</ui>
